spark/src/main/scala/io/spark/ddf/etl/TransformationHandler.scala

```diff
@@ -22,7 +22,12 @@ import io.ddf.etl.{TransformationHandler ⇒ CoreTransformationHandler}
 import io.ddf.exception.DDFException
 import io.spark.ddf.SparkDDF
 
+import java.io.PrintStream
+import com.google.api.client.util.LoggingOutputStream
+
 class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
+  // Rsession to use
+  val rsess = Rsession.newInstanceTry(new PrintStream(new LoggingOutputStream(mLog, false), true), null)
```

Comment:
@aht I tried to test with some R transforms, and it threw an error saying "task is not serializable" on the line where I construct Rsession inside the map(as in map-reduce). Do you know how we were able to do that with Rconnection? Mike suspected it was because of the System.out so I tried out another constructor for Rsession(val rsess = new Rsession(null, true)) and still getting the same error.